Assets/Scripts/HandTrigger.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+
+public class HandTrigger : MonoBehaviour, ThirdPersonInput.IInteractionActions
```

Comment:
Come nome, trigger mi fa pensare al trigger di un evento, magari la nomenclatura potrebbe essere <action>Controller.
Controller ha gia' una definizione sua comune nella programmazione, ma siccome non useremo MVC forse non sara' ambigua in questo progetto, poi volendo si puo fare refactoring in futuro (magari ActionHandler o InputHandler).
Quindi diventerebbe:
- ThirdPersonController -> MoveController
- HandTrigger -> TakeController/DragController/GrabController

Reply:
inizialmente era la mano che doveva entrare dentro alla scatola, da questo "handTrigger", poi ho cambiato tutto senza cambiare nome :P, comunque per il nome pernserò a qualcosa di più esplicativo